components/bsx/Offer/OfferTable.vue

```diff
@@ -182,7 +184,10 @@ export default class OfferTable extends mixins(OfferMixin) {
 .offer-title {
   line-height: 2rem;
   font-size: 1.2rem;
-  border-bottom: 2px solid hsl(0deg, 0%, 86%);
-  background: black;
+}
+ul {
+  &.pagination-list {
+    margin: 0 !important;
+  }
 }
```

Comment:
this will be global and might break something 😕

Reply:
Make sense! I have modified it.

Reply:
now it's better thank you